src/bytecodes/config.json

```diff
@@ -1,22 +1,22 @@
 [
   {
     "type": "ERC20Guild",
-    "bytecode_hash": "0xfc39d2eea44a6295b61acc49b32ab3779c21ef4aede2de9064ad93a24937bcd8",
+    "bytecode_hash": "0x439f8cef776487860ee78454d2ec1f1610e4db3b69ff66bbc5c91f1c4b991fb9",

```

Comment:
Why are these changing?

Reply:
ImplementationType hook wasn't working well so I update bytecodes and started working. Not sure if there was a change in the bytecodes.